types/mypads.pp

```diff
@@ -3,16 +3,11 @@
   {
     Optional['url']             => String,
     Optional['bindDN']          => String,
-    Optional['bindCredentials'] => String,
+    Optional['bindCredentials'] => String[1],
     Optional['searchBase']      => String,
     Optional['searchFilter']    => String,
     Optional['tlsOptions']      => Boolean,
     Optional['properties']      => Hash[String, String],
-    Optional['login']           => String,
```

Comment:
were those fields never used or didn't it work? I'm not sure if this is a bugfix or a backwards-incompatible change.

Reply:
Accordingly with doc it looks that that are into the key `properties`. And so they are not at the top level of the structure.

https://framagit.org/framasoft/Etherpad/ep_mypads/wikis/use-ldap-authentication